src/Components/CaseGrid/CaseThumb.js

```diff
@@ -93,33 +87,61 @@ const Img = styled.div`
   transform: scale(1.001);
 `
 
+const Description = styled.div`
+  font-size: ${props => props.full ? '30px' : '18px'};
+  & > * {
+    margin-top: 10px;
+  }
+  @media ${props => props.theme.media.lg} {
+    font-size: initial;
+  }
+`
+
 const Header = styled.h3`
   display: grid;
   align-content: center;
   margin: 0 calc(15px * 2.5);
-  font-size: ${props => props.full ? '30px' : '15px'};
+`
 
+const Logo = styled.img`
+  max-height: ${props => props.full ? '55px' : '35px'};
+  @media ${props => props.theme.media.sm} {
+    max-height: ${props => props.full ? '40px' : '35px'};
```

Comment:
In _sm view_, every `CaseThump` the same size. Therefore, the full sized `CaseThump` logos should not be larger in _sm view_.

Reply:
Also, please use `theme.spacing` again.